test/model/test_driver.py

```diff
@@ -14,6 +14,9 @@ def test_from_state():
     for msg in ws_msgs[1:]:
         msg = json.loads(msg)
         assert msg["type"] == "event"
-        driver.receive_event(msg["event"])
+        driver_event = driver_pkg.DriverEvent(
```

Comment:
These are not driver events. These are all events stored in the dump, which can be any source.

Reply:
I'll make a common Event class.